sync-core/src/main/java/com/cloudant/sync/datastore/PreparedAttachment.java

```diff
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2014 Cloudant, Inc. All rights reserved.
- *
+ * <p/>
```

Comment:
Do we need these para breaks? If we are concerned about how these comments like in javadoc they should probably all be wrapped in a <pre> block or something. The copyright headers probably all need revisiting soon to fix IBM/Cloudant attributions and other small variations.